src/admin/install/kunena.install.upgrade.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" ?>
+<!DOCTYPE comupgrade>
+<comupgrade version="1.0">
+    <upgrade>
+        <version version="5.2.0-ALPHA1"
+                 versiondate="2020-08-20"
+                 versionname="Internal">
+            <phpfile name="5.2.0-2020-08-20_updaterankstitle"/>
+        </version>
+        <version version="5.2.4-DEV"
+                 versiondate="2020-03-24"
+                 versionname="Internal">
+            <phpfile name="5.2.0-2021-03-24_configuration"/>
+        </version>
+        <version version="5.2.5-DEV"
+                 versiondate="2020-04-18"
+                 versionname="Internal">
+            <query mode="silenterror">ALTER TABLE `#__kunena_users` MODIFY banned datetime NULL DEFAULT '1000-01-01 00:00:00';</query>
+        </version>
+        <version version="5.2.6-DEV"
+                 versiondate="2021-06-12"
+                 versionname="Internal">
+            <query mode="silenterror">UPDATE `#__kunena_users_banned` SET `expiration` = '9999-12-31 23:59:59' WHERE expiration = '0000-00-00 00:00:00';</query>
+            <query mode="silenterror">UPDATE `#__kunena_users` SET `banned` = '9999-12-31 23:59:59' WHERE banned = '0000-00-00 00:00:00';</query>
+        </version>
+        <version version="6.0.0-RC99"
```

Comment:
RC99 ?

Reply:
I hadn't others ideas, i will need to change before the stable release